src/main/java/com/google/openthread/brski/Voucher.java

```diff
@@ -256,35 +261,37 @@ protected String getKeyName(String item) {
       new HashMap<String, Integer>() {
         {
           put(VOUCHER_REQUEST, VOUCHER_REQUEST_SID);
-          put(ASSERTION, get(VOUCHER_REQUEST) + 1 + 4);
-          put(CREATED_ON, get(VOUCHER_REQUEST) + 2 + 4);
-          put(DOMAIN_CERT_REVOCATION_CHECKS, get(VOUCHER_REQUEST) + 3 + 4);
-          put(EXPIRES_ON, get(VOUCHER_REQUEST) + 4 + 4);
-          put(IDEVID_ISSUER, get(VOUCHER_REQUEST) + 5 + 4);
-          put(LAST_RENEWAL_DATE, get(VOUCHER_REQUEST) + 6 + 4);
-          put(NONCE, get(VOUCHER_REQUEST) + 7 + 4);
-          put(PINNED_DOMAIN_CERT, get(VOUCHER_REQUEST) + 8 + 4);
-          put(PRIOR_SIGNED_VOUCHER_REQUEST, get(VOUCHER_REQUEST) + 9 + 4);
-          // put(PROXIMITY_REGISTRAR_CERT, get(VOUCHER_REQUEST) + 10 + 4);
-          put(PROXIMITY_REGISTRAR_SPKI, get(VOUCHER_REQUEST) + 11 + 21);
-          put(SERIAL_NUMBER, get(VOUCHER_REQUEST) + 12 + 2);
+          put(ASSERTION, get(VOUCHER_REQUEST) + 1);
+          put(CREATED_ON, get(VOUCHER_REQUEST) + 2);
+          put(DOMAIN_CERT_REVOCATION_CHECKS, get(VOUCHER_REQUEST) + 3);
+          put(EXPIRES_ON, get(VOUCHER_REQUEST) + 4);
+          put(IDEVID_ISSUER, get(VOUCHER_REQUEST) + 5);
+          put(LAST_RENEWAL_DATE, get(VOUCHER_REQUEST) + 6);
+          put(NONCE, get(VOUCHER_REQUEST) + 7);
+          put(PINNED_DOMAIN_CERT, get(VOUCHER_REQUEST) + 8);
+          put(PRIOR_SIGNED_VOUCHER_REQUEST, get(VOUCHER_REQUEST) + 9);
+          // put(PROXIMITY_REGISTRAR_CERT, get(VOUCHER_REQUEST) + 10);
```

Comment:
I think the "//" could be removed here as it looks correct; although it is not used by a Thread Device currently; and MUST NOT be used in the voucher request made to MASA.

Reply:
thanks, updated.